packages/events/defaultSubscribers/unbox.js

```diff
@@ -22,52 +22,62 @@ module.exports = {
   handlers: {
     "unbox:start": [
       function () {
+        if (this.quiet) return;
         this.logger.log(`${OS.EOL}Starting unbox...`);
         this.logger.log(`=================${OS.EOL}`);
       },
     ],
     "unbox:preparingToDownload:start": [
       function () {
+        if (this.quiet) return;
         this.spinner = this.ora("Preparing to download box").start();
       },
     ],
     "unbox:preparingToDownload:succeed": [
       function () {
+        if (this.quiet) return;
         this.spinner.succeed();
       },
     ],
     "unbox:downloadingBox:start": [
       function () {
+        if (this.quiet) return;
         this.spinner = this.ora("Downloading").start();
```

Comment:
hey, why don't you just wrap `this.ora(...).start()`? so that way you don't need to `if()` statement all over the place

Reply:
we'll have to wrap everything I think - then pass the action like "start", "fail", and "succeed" I think

Reply:
can just make a class, no? with `start`/`fail`/etc. methods

Reply:
At that point we're just pushing the problem somewhere else - I was imagining a function that had a switch statement. Then we only have to have one guard at the beginning of it